app/src/main/java/com/brandonnalls/mockmocklocations/OverrideSettingsSecure.java

```diff
@@ -10,14 +11,27 @@
 
 public class OverrideSettingsSecure implements IXposedHookLoadPackage {
     public void handleLoadPackage(final LoadPackageParam lpparam) throws Throwable {
+        // before API level 18, hook Settings.Secure.ALLOW_MOCK_LOCATION
+        if (Build.VERSION.SDK_INT < 18) {
```

Comment:
Is there a reason to limit this to SDK_INT < 18? ALLOW_MOCK_LOCATION isn't deprecated until API 23, and even then, apps can still use it.